examples/storage/storage.go

```diff
@@ -33,7 +34,24 @@ func Main(operation string, args []interface{}) interface{} {
 		return true
 	}
 
-	// TODO: storage.Find()
+	// Returns an array of key-value pairs with key that matched the passed value
+	if operation == "find" {
+		if checkArgs(args, 1) {
+			value := args[0].([]byte)
+			iter := storage.Find(ctx, value)
+
+			result := []string{}
+			for {
```

Comment:
Does `for iterator.Next(iter) {` form work?

Reply:
Yes, it does, fixed.